plugins/module_utils/azure_rm_common.py

```diff
@@ -947,7 +986,10 @@ def get_mgmt_svc_client(self, client_type, base_url=None, api_version=None, supp
             def _ansible_get_models(self, *arg, **kwarg):
                 return self._ansible_models
 
-            setattr(client, '_ansible_models', importlib.import_module(client_type.__module__).models)
+            try:
+                setattr(client, '_ansible_models', importlib.import_module(client_type.__module__).models)
+            except AttributeError:
+                setattr(client, '_ansible_models', importlib.import_module(client_type.__module__)._models)
```

Comment:
I had to do this when I moved to the newer azure.mgmt.rdbms library.  I think we shouldn't try and import models from the client module.

The azure client does
```
from azure.mgmt.rdbms import postgresql_flexibleservers

postgresql_flexibleservers.models.UserAssignedIdentity
```

Reply:
This is legacy code, it would be better to import models directly if possible

Reply:
Done, but I had to leave this since this method gets executed for all modules that use the get_mgmt_svc_client method that does the authentication.